mediaservices/azure-mediaservices-spring-boot-starter/src/main/java/com/microsoft/azure/autoconfigure/mediaservices/MediaServicesProperties.java

```diff
@@ -28,12 +28,12 @@
     /**
      * Media service account name.
      */
-    @NotEmpty
+    @NotEmpty(message = "Please Set accountName in application.properties")
```

Comment:
accountName might confuse user. How about azure.mediaservices.accountName?

Reply:
implemented the same